src/runtime/propensity-server.js

```diff
@@ -201,4 +246,8 @@ export class PropensityServer {
           return this.parsePropensityResponse_(response);
         });
   }
+
```

Comment:
Since this is a public API needs documentation.

Reply:
And lets rename to enableLoggingSwgEvents()

These are not google events.

Reply:
Done